sdk/notificationhubs/Azure.ResourceManager.NotificationHubs/src/Generated/NotificationHubAuthorizationRuleCollection.cs

```diff
@@ -21,28 +21,28 @@
 namespace Azure.ResourceManager.NotificationHubs
 {
     /// <summary>
-    /// A class representing a collection of <see cref="NamespaceNotificationHubAuthorizationRuleResource" /> and their operations.
-    /// Each <see cref="NamespaceNotificationHubAuthorizationRuleResource" /> in the collection will belong to the same instance of <see cref="NotificationHubResource" />.
-    /// To get a <see cref="NamespaceNotificationHubAuthorizationRuleCollection" /> instance call the GetNamespaceNotificationHubAuthorizationRules method from an instance of <see cref="NotificationHubResource" />.
+    /// A class representing a collection of <see cref="NotificationHubAuthorizationRuleResource" /> and their operations.
+    /// Each <see cref="NotificationHubAuthorizationRuleResource" /> in the collection will belong to the same instance of <see cref="NotificationHubResource" />.
+    /// To get a <see cref="NotificationHubAuthorizationRuleCollection" /> instance call the GetNotificationHubAuthorizationRules method from an instance of <see cref="NotificationHubResource" />.
     /// </summary>
-    public partial class NamespaceNotificationHubAuthorizationRuleCollection : ArmCollection, IEnumerable<NamespaceNotificationHubAuthorizationRuleResource>, IAsyncEnumerable<NamespaceNotificationHubAuthorizationRuleResource>
+    public partial class NotificationHubAuthorizationRuleCollection : ArmCollection, IEnumerable<NotificationHubAuthorizationRuleResource>, IAsyncEnumerable<NotificationHubAuthorizationRuleResource>
```

Comment:
The namespace is `NotificationHubs`, and we used `NotificationHub` here and `NotificationHubs` in some other places like `NotificationHubsSku`. It doesn't keep consistent, maybe we should use `NotificationHubs` as the namespace and `NotificationHub` as the prefix in the resources and models.

Reply:
Good catch, the `NotificationHubs` models are codegen auto-prepend models, fixed